ckanext/xloader/jobs.py

```diff
@@ -281,6 +297,13 @@ def get_url():
     logger.info('Express Load completed')
 
 
+def get_tmp_file(url):
+    # download the file to a tempfile on disk
```

Comment:
This comment should be kept with where it was called, rather than this function.